src/TMap.cpp

```diff
@@ -2008,7 +2008,7 @@ int TMap::createMapLabelID(int area)
     if (mapLabels.contains(area)) {
         QList<int> idList = mapLabels[area].keys();
         int id = 0;
-        while (id >= 0) {
```

Comment:
Technically this is trying to protect against overflow in the **signed** `int` - not perhaps an issue for an `int` but perhaps more relevant for something like `int_fast8_t` should it hit `INT_FAST8_MAX`...